src/Entity/BrandAwareInterface.php

```diff
@@ -7,12 +7,12 @@
 interface BrandAwareInterface
 {
     /**
-     * @return BrandInterface|null
+     * @return ProductsAwareInterface|null
```

Comment:
The `BrandInterface` extends the `ProductsAwareInterface` so I guess this should remain untouched

Reply:
For exemple someone who made his own `App\Entity\BrandInterface` with no extends from `Loevgaard\SyliusBrandPlugin\Entity\BrandInterface`, he want to be always compatible with the `Product` class he is linked to.
But this is just a little exemple, all the `*AwareInterface` should always have the smallest interface reference they could.

Reply:
Just to get my head around your thoughts: What would be a use case where a user uses this plugin, but creates his own `BrandInterface`?

Reply:
Did you see this question? :)

Reply:
I don't saw it sorry...
A use case would be someone who don't want the image part on the `Brand` entity and make his own `Entity` and `EntityInterface`

Reply:
Great, thanks!

Reply:
I guess Sylius solves this in their own components/bundles by having a base interface and then sub interface that extends i.e. the `ImagesAwareInterface`. But this is probably not an option here, so don't you think that we should impose the existence of products and images on a brand?